tpl/help/countjs-host.md

```diff
@@ -7,6 +7,8 @@ inside `<script>` tags. You won’t get any new features or other updates, but t
 To host `count.js` somewhere else just copy it from https://gc.zgo.at/count.js
 and adjust the URL in `src`.
 
+For convenience, the script is also published as an [npm package](https://www.npmjs.com/package/goatcounter) which can be copied into a public directory during your application's build process.
```

Comment:
I'm not sure if I understand what "can be copied into a public directory" means here?

Or to phrase it different: how would one use the npm package?

Reply:
The `count.js` source code is [included in the package](https://github.com/namoscato/goatcounter/blob/1cbe0026a5624f2bc092c4ef9527809d90f67b95/package.json#L26-L29) in addition to the type definitions, so instead of manually copying the file (per the existing sentence above), an application's build process could copy it from `./node_modules/goatcounter/public/count.js` into its public / webroot directory to serve locally.

Feel free to expand the suggested documentation update, or I could take another pass.

Reply:
No, the package just has to be referenced in [TSConfig `types`](https://www.typescriptlang.org/tsconfig/#types).

9b86465dd4d0c417aeeaded28e68e46bb8194004 expands documentation, but let me know if I can clarify anything else!

Reply:
Okay, I have it working locally by putting the count.d.ts in the public/ directory and adding `/// <reference path="./count.d.ts" />` to count.js

I haven't looked at the whole npm story yet, but that seems the best way to get it working in a simple npm-less setups? At the very least it's useful for me testing/changing the count.d.ts file going forward.